src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -1,58 +1,209 @@
+
+```console
+? Source Directory Path:  /
+? Distribution Directory Path: web-build
+? Build Command:  npx expo export:web
+? Start Command: npx expo start
+```
+
+</Callout>
+
+```console
+
+? Enter a name for the project (amplifiedtodo) 
+The following configuration will be applied:
+
+Project information
+| Name: amplifiedtodo
+| Environment: dev
+| Default editor: Visual Studio Code
+| App type: javascript
+| Javascript framework: react-native
+| Source Directory Path: src
+| Distribution Directory Path: /
+| Build Command: npm run-script build
+| Start Command: npm run-script start
+
+? Initialize the project with the above configuration? No
+? Enter a name for the environment dev
+? Choose your default editor: Visual Studio Code
+? Choose the type of app that you're building javascript
+Please tell us about your project
+? What javascript framework are you using react-native
+? Source Directory Path:  /
+? Distribution Directory Path: web-build
+? Build Command:  npx expo export:web
+? Start Command: npx expo start
+Using default provider  awscloudformation
+? Select the authentication method you want to use: AWS profile
+? Please choose the profile you want to use default
+
+...
+```
+</Block>
+
+<Block name="React Native CLI">
+
+```console
+
+? Enter a name for the project (amplifiedtodo) 
+The following configuration will be applied:
+
+Project information
+| Name: amplifiedtodo
+| Environment: dev
+| Default editor: Visual Studio Code
+| App type: javascript
+| Javascript framework: react-native
+| Source Directory Path: src
+| Distribution Directory Path: /
+| Build Command: npm run-script build
+| Start Command: npm run-script start
+
+? Initialize the project with the above configuration? Yes
+Using default provider  awscloudformation
+? Select the authentication method you want to use: AWS profile
+? Please choose the profile you want to use default
+
+...
 ```
 
 </Block>
 </BlockSwitcher>
 
-## Install Amplify libraries
+> Amplify CLI will infer the proper configuration based on the type of project Amplify is being initialized in where possible. Here it detected Create React App and provided the proper configuration for type of app, framework, source, distribution, build, and start options.
 
-Next, install the local Amplify dependencies. The directions here will depend on whether you are using Expo or the React Native CLI.
+When you initialize a new Amplify project, a few things happen:
 
-import all0 from "/src/fragments/start/getting-started/reactnative/getting-started-steps-ui.mdx";
+- It creates a top level directory called **amplify** that stores your backend definition. During the tutorial you'll add capabilities such as a GraphQL API and authentication. As you add features, the **amplify** folder will grow with infrastructure-as-code templates that define your backend stack. Infrastructure-as-code is a best practice way to create a replicable backend stack.
```

Comment:
I think this is also from the React tutorial and contains some inaccuracies about this tutorial. I might be wrong but I believe the RN tutorial never adds Auth directly and only touches GQL as a result of configuring DataStore.

Reply:
The current RN tutorial doesn't add auth, but I would like to make it so.

And DataStore is covered only in [JavaScript lib](https://docs.amplify.aws/lib/datastore/data-access/q/platform/js/) when it comes to js frameworks.

This would be another subject that I haven't researched yet:

* Can data store be used with js frameworks (I assume yes)?
* If yes, would it be smart to add switch blocks for API and DataStore?

Reply:
Ah, I see! I think this PR did not originally propose modifying the tutorial to be more closely aligned with React.

As for your question, regarding DataStore.. it seems, after speaking with some others, the general desire is to transition the tutorial towards the GraphQL/Auth one similar to React. so while

> Can data store be used with js frameworks (I assume yes)?

Yes it can...

> If yes, would it be smart to add switch blocks for API and DataStore?

We shouldn't need to do this. I think the two would result in pretty different approaches and would only introduce needless complexity to someone's introduction to Amplify.